kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -324,6 +324,7 @@ private TableInfo getTableInfo(TableId table, List<SinkRecord> records, Boolean
     List<com.google.cloud.bigquery.Schema> bigQuerySchemas = new ArrayList<>();
     Optional.ofNullable(readTableSchema(table)).ifPresent(bigQuerySchemas::add);
     for (SinkRecord record : records) {
+      logger.debug("Convert Schema for :"+record.topic()+"[p="+record.kafkaPartition()+"](o="+record.kafkaOffset()+")");
```

Comment:
A few comments:
- We may not actually do any conversion if the retriever returns `null`. This should probably be moved directly above the call to `convertRecordSchema`
- We're only converting the value schema here; key conversion takes place elsewhere. We should note this in the message here ("Converting value schema for...") and also add a logging message for key schema conversion where that takes place
- The message can be made more human-readable by eliminating shorthand syntax and becoming more verbose; something like "record with topic: <topic>, partition: <partition>, offset: <offset>"
- Slf4j marker syntax (i.e., `"{}"`) can be used instead of string concatenation

Reply:
Ok for marker syntax and more human readable.
For the first point I don't understand. I think this code was already there.

Reply:
This log line is replaced with a clearer error message in the `ConversionConnectException`.
Therefore, there is not really a need for this log line anymore.